apache2.yaml

```diff
@@ -83,6 +84,25 @@ pipeline:
 
   - uses: autoconf/make-install
 
+  - runs: |
+      set -x
+      # Modify User/Group and test to ensure changes are applied.
+      sed -ri \
+        -e 's!^(\s*User)\s+daemon\s*$!\1 www-data!g' \
+        -e 's!^(\s*Group)\s+daemon\s*$!\1 www-data!g' \
+        "${{targets.destdir}}"/etc/httpd.conf
+
+      grep -E '^\s*User www-data$' "${{targets.destdir}}"/etc/httpd.conf;
+      grep -E '^\s*Group www-data$' "${{targets.destdir}}"/etc/httpd.conf;
+
+      # Modify CustomLog/ErrorLog/TransferLog (it's ok it fails, no need to test).
```

Comment:
If it is OK if it fails, then why do we do it?
Given the option between 
1. software that isn't necessary and only _might_ do something
2. no software

I will pick 2.